src/components/NetworkListTable.tsx

```diff
@@ -45,7 +52,39 @@ const NetworkListTable: FC<Props> = ({ onFailure, devices, instance }) => {
     },
   ];
 
-  const networksRows = Object.entries(devices ?? {})
+  const lxdNetworks = Object.entries(devices ?? {})
+    .map(([_, networkDevice]) => {
+      if (networkDevice.type !== "nic") {
+        return null;
+      }
+
+      const network = networks.find(
+        (item) =>
+          item.name === networkDevice.network ||
+          item.name === networkDevice.parent,
+      );
+
+      if (!network) {
+        return null;
+      }
+
+      return network;
+    })
+    .filter((network) => network !== null);
+
+  const shouldDisplayAclsColumn = lxdNetworks.some(
+    (t) => t.config["security.acls"],
+  );
+
+  if (shouldDisplayAclsColumn) {
+    networksHeaders.push({
+      content: "ACLs",
+      sortKey: "acls",
+      className: "u-text--muted u-hide--small u-hide--medium",
+    });
+  }
```

Comment:
Adding this column leaves not enough space for the mac address to be fully visible. I think we can reduce the width of the Interface column slightly to make enough space for it.